src/SFA.DAS.EmployerAccounts.Jobs/StructureMapJobActivator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Azure.WebJobs.Host;
+using StructureMap;
+
+namespace SFA.DAS.EmployerAccounts.Jobs
```

Comment:
Can this file be moved to the `DependencyResolution` folder?

Reply:
should really be in a Nuget package, such as SFA.DAS.StructureMap.WebJob ???